File: app/src/main/java/org/wikipedia/feed/accessibility/AccessibilityCardClient.java
package org.wikipedia.feed.accessibility;

import org.wikipedia.dataclient.WikiSite;
import org.wikipedia.feed.dataclient.DummyClient;
import org.wikipedia.feed.model.Card;

public class AccessibilityCardClient extends DummyClient {
    @Override public Card getNewCard(WikiSite wiki) {
        return new AccessibilityCard();
    }
}

File: app/src/main/java/org/wikipedia/feed/accessibility/AccessibilityCardClient.kt
package org.wikipedia.feed.accessibility

import org.wikipedia.dataclient.WikiSite
import org.wikipedia.feed.dataclient.DummyClient
import org.wikipedia.feed.model.Card

class AccessibilityCardClient : DummyClient() {

    override fun getNewCard(wiki: WikiSite?): Card {
        return AccessibilityCard()
    }
}
